README.md

```diff
@@ -40,6 +40,7 @@ Table of Contents
         * [Uploaded File Storage](#uploaded-file-storage)
         * [Deleting Uploaded Files](#deleting-uploaded-files)
         * [S3 File Retention Policies](#s3-file-retention-policies)
+    * [Address Validation](#Address Validation)
```

Comment:
I just noticed that this link format is incorrect. The link should be `#address-validation` for this to work. 

The way I found this was by using 'View file', so it rendered the markup in GitHub.

![Screenshot 2023-01-26 at 11 38 02 AM](https://user-images.githubusercontent.com/7966375/214894642-f3f7ef1f-210c-4e38-b4a1-c5d3c8388104.png)

 Another issue jumped out and I'll mark it below.